### PR TITLE
Android native logs

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add android Native Logs. IOS logs will be added in a later PR.

#### Related PR
https://github.com/mattermost/react-native-turbo-log/pull/6

#### Ticket Link
Part of https://mattermost.atlassian.net/browse/MM-61970

#### Release Note
```release-note
Android: App logs will show information from the native side
```

### CODE REVIEW

File: patches/@mattermost+react-native-turbo-log+0.4.0.patch
Comment (enahum) [inline patches/@mattermost+react-native-turbo-log+0.4.0.patch:1]: before merging this PR, we should update the library and remove this patch

**Review by enahum — APPROVED** [review]
This looks good, some changes needed in the library and then update this once the library npm package is released

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@mattermost/react-native-emm": "1.5.0",
     "@mattermost/react-native-network-client": "1.7.3",
     "@mattermost/react-native-paste-input": "0.8.0",
-    "@mattermost/react-native-turbo-log": "0.4.0",
+    "@mattermost/react-native-turbo-log": "github:mattermost/react-native-turbo-log#d8ddf5e7974546aff3e83b2c563907eb609ac5f4",
```
Comment (larkox) [inline package.json:27]: @enahum Let's use the github commit until we have the iOS version in place and we bump the version of the library.

**Comment by larkox** [comment]
@yasserfaraazkhan On a dev build, by defualt, the react native part won't create logs, so test this with a production build.

The idea is that now you will have new logs, based on what happens on the native side (specially, when receiving notifications). Also, make sure that iOS has no relevant changes (no sudden crashes and the logs work just fine (just not the native part)).

**Review by enahum — APPROVED** [review]

**Comment by enahum** [comment]
going to merge this.

**Pull request merged**